src/utils/convert.ts

```diff
@@ -94,3 +94,13 @@ export function parsePartitionBytes32Value(value: string): Partition {
       throw new Error('Partition not recognized');
   }
 }
+export function parsePermBytes32Value(value: string): Perm {
+  switch (bytes32ToString(value)) {
+    case 'ADMIN':
```

Comment:
Here the case values can be the permission values as well

i.e. `case Perm.Admin`